pkg/kubectl/cmd/config/create_authinfo.go

```diff
@@ -260,6 +376,27 @@ func (o *createAuthInfoOptions) complete(cmd *cobra.Command, out io.Writer) erro
 		o.authProviderArgsToRemove = removePairs
 	}
 
+	execArgs, err := cmd.Flags().GetStringSlice(flagExecArg)
+	if err != nil {
+		return fmt.Errorf("Error: %s", err)
+	}
+	if len(execArgs) > 0 {
+		o.execArgs = execArgs
```

Comment:
does this mean we cannot unset execargs? we only assign to o.execArgs if len is greater than zero, and we only set `modifiedAuthInfo.Exec.Args = o.execArgs` if `o.execArgs` is non-nil.

Reply:
Right... I've update the pr, and now if a user provides --exec-command, then the arguments will be automatically reset.